src/ldf-client-ui.js

```diff
@@ -114,6 +114,7 @@ require('leaflet/dist/images/marker-shadow.png');
       // Initialize map
       this.map = L.map($map.get(0)).setView([52.517987721, 6.116665362], 8);
       this.mapLayer = L.geoJSON().addTo(this.map);
+      this.trialLayerGroup = L.layerGroup().addTo(this.map);
```

Comment:
What does `trialLayerGroup` mean? Can we have a more descriptive name?

Reply:
This is the LayerGroup variable am adding my layers to. sorry named it trial as I was testing it out